gunicorn/arbiter.py

```diff
@@ -582,8 +580,6 @@ def spawn_worker(self):
             self.cfg.post_fork(self, worker)
             worker.init_process()
             sys.exit(0)
-        except SystemExit:
```

Comment:
This change is incorrect, because there is a bare `except` clause in line 589. We need to specifically catch and reraise `SystemExit`.

Reply:
got it, should be fine now as we're going to ignore `try-except-raise`.